### PR TITLE
Added styles for features list on landing page

### PR DESCRIPTION
**Pull request by bartaz**
screen shot 2017-02-15 at 11 57 19

### CODE REVIEW

File: src/common/components/vanilla/list/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import ListDivided from './list-divided';
+import ListDividedState from './list-divided-state';
```
Comment (bartaz) [inline src/common/components/vanilla/list/index.js]: List components based on https://github.com/ubuntudesign/vanilla-design/blob/master/Lists/lists.png

Comment (katiefenn) [inline src/common/components/vanilla/list/index.js]: Why not use a `<ul>` as the encapsulating element?

File: src/common/components/vanilla/list/list-divided-state.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React, { PropTypes } from 'react';
+import styles from './list-divided-state.css';
+
+export default function ListDividedState(props) {
+  const className = props.className || '';
+
+  return (
+    <div className={`${styles.listDividedState} ${className}`}>
+      { props.children }
+    </div>
```
Comment (katiefenn) [inline src/common/components/vanilla/list/list-divided-state.js]: Why not use a <ul> as the encapsulating element?

File: src/common/components/vanilla/list/list-divided.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React, { PropTypes } from 'react';
+import styles from './list-divided.css';
+
+export default function ListDivided(props) {
+  const className = props.className || '';
+
+  return (
+    <div className={`${styles.listDivided} ${className}`}>
+      { props.children }
+    </div>
```
Comment (katiefenn) [inline src/common/components/vanilla/list/list-divided.js]: Why not use a <ul> as the encapsulating element?

Comment (katiefenn) [inline src/common/components/vanilla/list/list-divided.js]: ```</div>```

File: src/common/components/vanilla/list/list-divided-state.js (hunk 2 above)
Comment (bartaz) [inline src/common/components/vanilla/list/list-divided-state.js]: Oh, good catch - that was my intention.
Copy/paste error ;)

**Pull request merged**